app/controllers/application_controller.rb

```diff
@@ -75,6 +75,24 @@ def find_platform(param = :id)
   def find_project
     @project = Project.find_with_includes!(params[:platform], params[:name], [:repository, :versions])
     @color = @project.color
+  rescue ActiveRecord::RecordNotFound
+    raise if params[:name].blank? || params[:platform].downcase != "go"
+
+    resolved_name = PackageManager::Go.resolved_name(params[:name])
+    if resolved_name != params[:name] && Project.known?(params[:platform], resolved_name)
+      redirect_to(
```

Comment:
Should we redirect or just use it as `project` at this point?

Reply:
I was thinking redirect since it felt strange to serve data that would say "project name: foo" when the url said "project name: bar" - open to discussion on that though.